package.json

```diff
@@ -27,6 +27,7 @@
   "main": "lib/stripe.js",
   "types": "types/2022-08-01/index.d.ts",
   "devDependencies": {
+    "@istanbuljs/nyc-config-typescript": "^1.0.2",
```

Comment:
why this particular config?

Reply:
Ah good catch, I preemptively added that when I was thinking about test coverage on the TS files, but it's unnecessary because our tests still use the compiled JS in `lib/` Will remove!